cmd/peapod-to-fstree/main.go

```diff
@@ -46,7 +46,14 @@ func (s *onlyObjectsWithContainersStorage) Iterate(objHandler func(oid.Address,
 		}
 
 		return objHandler(addr, data, id)
-	}, nil)
+	}, func(addr oid.Address, err error) error {
+		_, found := s.containers[addr.Container()]
+		if !found {
+			log.Printf("skipping corrupted object without container '%s': %v", addr, err)
+			return nil
+		}
+		return err
```

Comment:
shouldnt we return nil to continue storage traverse?

Reply:
seems like no https://github.com/nspcc-dev/neofs-node/pull/3306#discussion_r2046881011